config/database.yml

```diff
@@ -21,25 +21,25 @@ default: &default
 
 development:
   <<: *default
-  database: user_api_development
+  database: <%= ENV['USER_DATABASE'] %>
```

Comment:
No need to use ENV for database name

Reply:
In most of the projects they include database name as well, anyway I will remove it.